src/Test.UnitTests.BinSkim.Driver/MultithreadedAnalyzeCommandTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.IL;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.BinSkim.Rules
+{
+    public class MultithreadedAnalyzeCommandTests
+    {
+        [Fact]
+        public void MultithreadedAnalyzeCommand_ReturnCommonPathRootFromTargetSpecifiersIfOneExists()
```

Comment:
>MultithreadedAnalyzeCommand_ReturnCommonPathRootFromTargetSpecifiersIfOneExists [](http://example.com/codeflow?start=20&length=79)

add tests for double slashes in the path.
add test for equivalent paths:

c:\mytest\..\*.dll
c:\*.dll